app/assets/src/api/bulk_downloads.js

```diff
@@ -8,3 +8,8 @@ export const createBulkDownload = bulkDownload =>
     sample_ids: bulkDownload.sampleIds,
     params: bulkDownload.fields,
   });
+
+export const getBulkDownloads = () => get("/bulk_downloads.json");
+
+export const getBulkDownloadDetails = bulkDownloadId =>
```

Comment:
Could call it just `getBulkDownload`... more in like with the REST action `show`.